README.md

```diff
@@ -62,6 +62,127 @@ Please the [README.md](deploy/README.md) in the `deploy` folder for instructions
 npm test
 ```
 
+##### Gas usage report
```

Comment:
This information is hard to analyse especially if you are not familiar with the bridge contracts code execution flow. Please take a look at the review comment to consider more practical example of the report that it make sense to add to the README.